docs/docs/getting-started/03-installation-and-launch.md

```diff
@@ -85,4 +103,9 @@ first time launching a configuration, the AICA Core software image and any addit
 from the registry and bundled into the specific runtime image. Depending on network speeds, this may take a while. After
 the image has been built, launching the configuration a second time will be much faster as the results are cached.
 
-![aica-launcher-studio](./assets/aica-launcher-studio.png)
\ No newline at end of file
+![aica-launcher-studio](./assets/aica-launcher-studio.png)
+
+## Manual installation and launch
+
+For advanced users or users that deal with headless machines, there exists the option to [manually perform the login,
```

Comment:
Happy to rephrase that if desired

Reply:
Why did you choose to put that in brackets? I think wording is fine but would just remove the brackets

Reply:
it's a way to format a link (`[text](link)`) 😉

Reply:
ok perfect then :) disregard Dominic